### PR TITLE
chore: upgrade kurtosis feature in cloud instance in experiment mode

### PR DESCRIPTION
**Pull request by leoporoli**
## Description
Adding the UI component to upgrade Kurtosis in the cloud instance

## Is this change user facing?
NO

## References (if applicable)
This is part of the `upgrade Kurtosis from the UI project`

### CODE REVIEW

File: enclave-manager/web/packages/app/src/emui/enclaves/components/KurtosisUpgrader.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        if (instanceStatus === "running") {
+          setUpgradeStatus(UpgradeStatus.SUCCESS);
+          clearInterval(interval);
+        }
+      } catch (error) {
+        console.error(`Error occurred getting the cloud instance config. ${error}`);
+        setUpgradeStatus(UpgradeStatus.ERROR);
+        clearInterval(interval);
+      }
+    }, 2000);
```
Comment (laurentluce) [inline enclave-manager/web/packages/app/src/emui/enclaves/components/KurtosisUpgrader.tsx]: The upgrade operation is in the order of tens of seconds I am guessing.  Can we check every 10 sec to not overload the cloud backend.

File: enclave-manager/web/packages/app/src/emui/experiments/Experiments.tsx
Comment (laurentluce) [inline enclave-manager/web/packages/app/src/emui/experiments/Experiments.tsx:1]: Neat!

**Review by laurentluce — APPROVED** [review]

File: enclave-manager/web/packages/app/src/emui/enclaves/components/KurtosisUpgrader.tsx (hunk 1 above)
Comment (leoporoli) [reply enclave-manager/web/packages/app/src/emui/enclaves/components/KurtosisUpgrader.tsx]: yes, we can! I'll do the change

Comment (leoporoli) [reply enclave-manager/web/packages/app/src/emui/enclaves/components/KurtosisUpgrader.tsx]: done

**Pull request merged**